### ISSUE TITLE
(IOU: 0.000000, GIOU: 0.000000), Class: 0.000000, Obj: 0.000000... can not detect anything after train! Someone helps me, plz

### ISSUE THREAD

**Issue by anjipham** [pjreddie/darknet#2104]
![image](https://user-images.githubusercontent.com/62320446/78710950-6524cd80-7940-11ea-9518-100a398ec571.png)
![image](https://user-images.githubusercontent.com/62320446/78710977-6fdf6280-7940-11ea-988d-24400aa3474b.png)

whats wrong?
Someone help

[net]
# Testing
#batch=1
#subdivisions=1
# Training
batch=64
subdivisions=16
width=416
height=416
channels=3
momentum=0.9
decay=0.0005
angle=0
saturation = 1.5
exposure = 1.5
hue=.1

learning_rate=0.001
burn_in=1000
max_batches = 10000
policy=steps
steps=8000,9000
scales=.1,.1

**Comment by anjipham** [comment]
anyone help, plz!!!

**Comment by Devin97** [comment]
Did you convert your labels to yolo format?

**Comment by anjipham** [comment]
thanks @Devin97 for reply. yeah, i made mistake as convert annotation!! ^^
